src/refract/TypeQueryVisitor.h

```diff
@@ -90,11 +90,11 @@ namespace refract
             }
 
             TypeQueryVisitor tq;
-            tq.visit(*e);
+            Visit(tq, *e);
 
             E type;
             TypeQueryVisitor eq;
-            type.content(eq);
+            Visitby(eq, type);
```

Comment:
We discuss it with Vilda. Finel solutio is this.
Reason:
`Visit(visitor, element)` - shortcut for `visitor.visit(element)` in meaning "visitor visiting element"
`VisitBy(element, visitor)` - shortcut `element.content(visitor)` in meaning "Element visited by visitor"